src/shared/components/modlog.tsx

```diff
@@ -609,7 +612,7 @@ export class Modlog extends Component<any, ModlogState> {
                 <option value={ModlogActionType.ModAdd}>Adding Mod to Site</option>
                 <option value={ModlogActionType.ModBan}>Banning From Site</option>
               </select>
-              {!this.state.site_view.site.hide_modlog_mod_names && <select
+              {this.state.siteRes.site_view.isSome() && !this.state.siteRes.site_view.unwrap().site.hide_modlog_mod_names && <select
```

Comment:
Use a match, there should be lots of examples, especially in search.tsx 

In general if you see an unwrap, it means there's a better way to do it, because an unwrap could crash things.